supervisor.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+set -e
+
+trap 'kill %1; kill %2' ERR EXIT
+
+cmd=$PWD/transfer2go
```

Comment:
Rishi, please don't take me wrong (as being picky), I just want to show you the reason. What you did is still not optimal since it means that you need to run supervisor.sh script from the area where transfer2go reside. Let's assume that this is not the case, e.g. transfer2go may be under a different area. I want supervisor.sh to be called with given executable, e.g.
```
supervisor.sh /path/transfer2go <args>
```